src/i_oplmusic.c

```diff
@@ -359,7 +359,7 @@ static unsigned int last_perc_count;
 // Configuration file variable, containing the port number for the
 // adlib chip.
 
-static char *snd_dmxoption = "-opl3"; // [crispy] default to OPL3 emulation
+static char *snd_dmxoption = "-opl3-reverse"; // [crispy] default to OPL3 emulation
```

Comment:
Please revert this. I've had this in Crispy for a while and people reported issues when the music didn't sound as they were used to anymore. 

Sometimes "technically accurate" and "Vanilla" have contrary means, and in these cases we tend to decide for "Vanilla".